base_report_auto_create_qweb/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,11 @@
+When creating a report in Settings > Technical > Actions > Reports or
```

Comment:
this menu doesn't exist anymore in 14.0, please replace it by the following:
`Settings > Technical > Actions > Reports`

Reply:
it'ok, I misread it!